### PR TITLE
JW-1314 dns cleanup

### PR DESCRIPTION
**Pull request by katrybacka**

### CODE REVIEW

**Comment by katrybacka** [comment]
Janie, przetestuj proszę

File: src/dns/bind/named_config.cc [hunk 1]
```diff
@@ -128,8 +127,9 @@ void NamedConfig::DelZone(const Subnet &subnet, const VirtualDnsConfig *vdns) {
 }
 
 void NamedConfig::UpdateNamedConf(const VirtualDnsConfig *updated_vdns) {
+    #ifndef _WIN32
     CreateNamedConf(updated_vdns);
-    //WINDOWS-TEMP sync();
+    sync();
```
Comment (kaweue) [inline src/dns/bind/named_config.cc]: Could we add sync()  in windowssubs ?

**Review by kaweue — CHANGES_REQUESTED** [review]

**Review by jablonskim — APPROVED** [review]

File: src/dns/bind/named_config.cc [hunk 2]
```diff
@@ -129,7 +128,9 @@ void NamedConfig::DelZone(const Subnet &subnet, const VirtualDnsConfig *vdns) {
 
 void NamedConfig::UpdateNamedConf(const VirtualDnsConfig *updated_vdns) {
     CreateNamedConf(updated_vdns);
-    //WINDOWS-TEMP sync();
+    #ifndef _WIN32
```
Comment (kaweue) [inline src/dns/bind/named_config.cc]: Please move it a bit to the left.

**Review by kaweue — CHANGES_REQUESTED** [review]

**Review by kaweue — APPROVED** [review]

File: src/dns/bind/named_config.cc [hunk 3]
```diff
@@ -129,7 +128,9 @@ void NamedConfig::DelZone(const Subnet &subnet, const VirtualDnsConfig *vdns) {
 
 void NamedConfig::UpdateNamedConf(const VirtualDnsConfig *updated_vdns) {
     CreateNamedConf(updated_vdns);
-    //WINDOWS-TEMP sync();
+#ifndef _WIN32
+    sync();
```
Comment (mclapinski) [inline src/dns/bind/named_config.cc:132]: Try `::sync()` and implement `sync()` in windowsstubs

File: src/dns/test/dns_bind_test.cc [hunk 4]
```diff
@@ -231,8 +231,9 @@ TEST_F(DnsBindTest, Config) {
 }
 #endif
 
+#ifndef _WIN32
 // TODO(WINDOWS): JW-1168: NamedConfig is not used in the Agent
```
Comment (mclapinski) [inline src/dns/test/dns_bind_test.cc]: You kept this comment...

File: src/dns/test/dns_bind_test.cc [hunk 5]
```diff
@@ -519,8 +520,7 @@ TEST_F(DnsBindTest, DISABLED_Reordered) {
     }
 }
 
-// TODO(WINDOWS): JW-1168: NamedConfig is not used in the Agent
```
Comment (mclapinski) [inline src/dns/test/dns_bind_test.cc:522]: ... but you removed this one.

**Review by mclapinski — CHANGES_REQUESTED** [review]

File: src/dns/test/dns_bind_test.cc (hunk 4 above)
Comment (katrybacka) [reply src/dns/test/dns_bind_test.cc]: I ifndefed two methods, but I should move this comment one line above

**Review by mclapinski — APPROVED** [review]

**Pull request merged**